DAG-insert-elk-sirene.py

```diff
@@ -50,7 +51,7 @@
     default_args=default_args,
     schedule_interval="0 23 10 * *",
     start_date=days_ago(10),
-    dagrun_timeout=timedelta(minutes=60 * 8),
+    dagrun_timeout=timedelta(minutes=60 * 20),
```

Comment:
Hum. Can we go back to 8 hours max please

Reply:
Yes :)